### PR TITLE
Add time function and function to add int

### PR DESCRIPTION
**Pull request by MartinPetkov**
Useful for generating the current time or expiry dates (via `(Time).AddDate`).

### CODE REVIEW

File: internal/template/funcmap.go [hunk 1]
```diff
@@ -28,6 +29,8 @@ var funcMap = map[string]interface{}{
 	"hclField":     hclField,
 	"replace":      replace,
 	"resourceName": resourceName,
+	"now":          time.Now,
```
Comment (xingao267) [inline internal/template/funcmap.go:32]: what's the use case for these?

File: internal/template/funcmap.go [hunk 2]
```diff
@@ -111,3 +114,12 @@ func resourceName(m map[string]interface{}, key string) (string, error) {
 func replace(s, old, new string) string {
 	return strings.Replace(s, old, new, -1)
 }
+
+// add adds a list of integers together
+func add(nums ...int) int {
```
Comment (umairidris) [inline internal/template/funcmap.go:119]: sum

File: internal/template/funcmap.go (hunk 1 above)
Comment (MartinPetkov) [reply internal/template/funcmap.go:32]: It would be useful for generating expiry dates.

File: internal/template/funcmap.go (hunk 2 above)
Comment (MartinPetkov) [reply internal/template/funcmap.go]: Removed.

**Pull request merged**